pcsx2/gui/AppCoreThread.cpp

```diff
@@ -264,12 +264,23 @@ static int loadGameSettings(Pcsx2Config& dest, const Game_Data& game) {
 
 
 	if (game.keyExists("mvuFlagSpeedHack")) {
-		bool vuFlagHack = game.getInt("mvuFlagSpeedHack") ? 1 : 0;
+		bool vuFlagHack = !!game.getInt("mvuFlagSpeedHack");
 		PatchesCon->WriteLn("(GameDB) Changing mVU flag speed hack [mode=%d]", vuFlagHack);
 		dest.Speedhacks.vuFlagHack = vuFlagHack;
 		gf++;
+		
+		//The mVU flag speed hack cause a speed issue on some Katamari games.
```

Comment:
These game specific comments aren't necessary and don't really fit inside the core code.